Asterix/wfsc/estimator.py

```diff
@@ -99,6 +99,8 @@ def __init__(self,
         if self.technique == "perfect":
             self.is_focal_plane = True
             self.is_complex = True
+            if self.polychrom == 'broadband_pwprobes':
+                raise Exception("cannot use polychrom='broadband_pwprobes' in perfect mode")
```

Comment:
Might be better to be explicit and use a `ValueError` here instead of the Exception().